label.tf

```
@@ -0,0 +1,79 @@
+module "label" {
+  source              = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.11.1"
+  attributes          = ["${var.attributes}"]
+  namespace           = "${var.namespace}"
+  environment         = "${var.environment}"
+  stage               = "${var.stage}"
+  delimiter           = "${var.delimiter}"
+  name                = "${var.name}"
+  tags                = "${var.tags}"
+  additional_tag_map  = "${var.additional_tag_map}"
+  regex_replace_chars = "${var.regex_replace_chars}"
+  label_order         = "${var.label_order}"
+  context             = "${var.context}"
+}
+
+variable "additional_tag_map" {
+  type        = "map"
+  default     = {}
+  description = "Additional tags for appending to each tag map"
+}
+
+variable "label_order" {
+  type        = "list"
+  default     = []
+  description = "[Label Module] The naming order of the id output and Name tag"
```

```suggestion
  description = "The naming order of the id output and Name tag"
```

I think we don't need to use `[Label Module]` in the descriptions, it makes it more difficult to read

---

I understand. I was wanting a way to categorise the label modules inputs, so that they could be separated from the dynamic subnets module variables.
Is there another way we could categorise it in the readme @aknysh?

---

@Jamie-BitFlight 
The README generator reads all descriptions and puts them into  README (so no fast way to categorize them).
I think we don't need to categorize with `[Label Module]`, the users of the module (and other CloudPosse modules) are already familiar with the `label` attributes, so there is no reason to separate them from other module's variables